bin/omakub-font

```diff
@@ -1,18 +1,15 @@
-# Pick a preconfigured theme
 FONT=$(gum choose "Cascadia Mono" "Fira Mono" "JetBrains Mono" "Meslo" --header "Choose your programming font:" --height 6 | tr '[:upper:]' '[:lower:]' | sed 's/ /-/g')
 
-[ ! -n "$FONT" ] && exit 0
+[ -z "$FONT" ] && exit 0
 
-if [ "$FONT" == "cascadia-mono" ]; then
-	NERD_FONT="CaskaydiaMono Nerd Font"
-elif [ "$FONT" == "fira-mono" ]; then
-	NERD_FONT="FiraMono Nerd Font"
-elif [ "$FONT" == "jetbrains-mono" ]; then
-	NERD_FONT="JetBrainsMono NFM"
-elif [ "$FONT" == "meslo" ]; then
-	NERD_FONT="MesloLGLDZ Nerd Font"
-fi
+declare -A NERD_FONTS=(
+  ["cascadia-mono"]="CaskaydiaMono Nerd Font"
+  ["fira-mono"]="FiraMono Nerd Font"
+  ["jetbrains-mono"]="JetBrainsMono NFM"
+  ["meslo"]="MesloLGLDZ Nerd Font"
+)
```

Comment:
Could see using `case` instead. Prefer syntax over a data type for this usage.

Reply:
fixed